.gitignore

```diff
@@ -0,0 +1,2810 @@
+
```

Comment:
A .gitignore file is not needed so you can delete this

Reply:
done

Reply:
fixed the bug where white space was being counted as character

Reply:
fixed the most frequent words casing problem